lib/statemanager.js

```diff
@@ -449,6 +449,20 @@ StateManager.prototype.sign = function(address, dataToSign) {
   return utils.toRpcSig(sgn.v, sgn.r, sgn.s);
 };
 
+StateManager.prototype.personal_sign = function(address, dataToSign) {
+  var account = this.accounts[to.hex(address).toLowerCase()];
+
+  if (!account) {
+    throw new Error("cannot sign data; no private key");
+  }
+
+  return sigUtil.personalSign(account.secretKey, { data: dataToSign });
+};
+
+StateManager.prototype.personal_ecRecover = function(dataThatWasSigned, signature) {
+  return sigUtil.recoverPersonalSignature({ data: dataThatWasSigned, sig: signature });
```

Comment:
I think it's not good to use the abbreviation. I would rename **sig** to **signature**.

Reply:
ok